jsonschema/schema_test.go

```diff
@@ -131,6 +131,16 @@ func (s *SchemaSuite) TestDefaults(c *C) {
 				},
 			},
 		},
+		{
+			schema: `{
```

Comment:
what does this test cover? what was the behavior for this test before your change?